.template/lib/utils/wrappers/permission_wrapper.dart

```diff
@@ -10,7 +9,6 @@ abstract class PermissionWrapper {
   Future<bool> isCameraPermissionPermanentlyDenied();
 }
 
-@Singleton(as: PermissionWrapper)
```

Comment:
FYI: no `injectable` integrated yet, should update it later

Reply:
👍